addons/website/static/src/js/website.contentMenu.js

```diff
@@ -236,6 +238,7 @@ var EditMenuDialog = widget.Dialog.extend({
     add_menu: function () {
         var self = this;
         var dialog = new MenuEntryDialog(this, {menu_link_options: true}, undefined, {});
+        dialog.$modal.addClass('o_contentMenu_link_modal');
```

Comment:
Sorry but I think we should remove this as well. I saw your padding rules so that the link dialog is smaller than the menu dialog... I think more code for this is not worth it. If we want this behavior, it should be handled by the Dialog class and this should be the subject of another task (it could actually be nice: if a dialog has a parent dialog, then it is styled differently automatically).